### DISCUSSION TITLE
Which linux OS is best for frigate ?

### DISCUSSION THREAD

**Discussion by peter247** [blakeblackshear/frigate#3020] [General]
Hi , I've been testing out a mele quieter2 fanless PC which uses a j4125 cpu for suitability to run frigate without a TPU .
So far so good it's running 3 cameras , 2 x h264 and a h264 camera at about 35% cpu without any detection , but running this on proxmox I'm getting a full computer lock up every day which I think is related to frigate and hardware acceleration ?
Because I've run VMs on it for days without problems .
So thinking of just using a better suited OS just to run frigate and my be a samba server .  

Which OS is more likely the stable ?

**Comment by blakeblackshear** [comment]
I recommend Debian Bullseye.
